crud/src/main/java/com/redhat/lightblue/crud/LightblueHealth.java

```diff
@@ -0,0 +1,32 @@
+package com.redhat.lightblue.crud;
+
+/**
+ *
+ */
+public class LightblueHealth {
```

Comment:
I think `CRUDHealth` is probably a better name for this–it continues the pattern of the rest of the return types in CRUDController, so this is more discoverable / less surprising in my opinion.